spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,220 @@
+description = 'Spring Integration Hazelcast Support'
+
+apply plugin: 'java'
+apply from: "${rootProject.projectDir}/publish-maven.gradle"
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+group = 'org.springframework.integration'
+
+repositories {
+	maven { url 'http://repo.spring.io/libs-milestone' }
+}
+
+sourceCompatibility = targetCompatibility = 1.7
+
+ext {
+	hazelcastVersion = '3.4.1'
+	jacocoVersion = '0.7.0.201403182114'
+	springIntegrationVersion = '4.1.2.RELEASE'
+
+	idPrefix = 'hazelcast'
+
+	linkHomepage = 'https://github.com/spring-projects/spring-integration-extensions'
+	linkCi = 'https://build.spring.io/browse/INTEXT'
+	linkIssue = 'https://jira.spring.io/browse/INTEXT'
+	linkScmUrl = 'https://github.com/spring-projects/spring-integration-extensions'
+	linkScmConnection = 'https://github.com/spring-projects/spring-integration-extensions.git'
+	linkScmDevConnection = 'git@github.com:spring-projects/spring-integration-extensions.git'
+}
+
+eclipse.project.natures += 'org.springframework.ide.eclipse.core.springnature'
+
+sourceSets {
+	test {
+		resources {
+			srcDirs = ['src/test/resources', 'src/test/java']
+		}
+	}
+}
+
+configurations {
+	jacoco //Configuration Group used by Sonar to provide Code Coverage using JaCoCo
+}
+
+dependencies {
+	compile "org.springframework.integration:spring-integration-core:$springIntegrationVersion"
+	compile "com.hazelcast:hazelcast:$hazelcastVersion"
+
+	testCompile "org.springframework.integration:spring-integration-test:$springIntegrationVersion"
+
+	jacoco "org.jacoco:org.jacoco.agent:$jacocoVersion:runtime"
+}
+
+// enable all compiler warnings; individual projects may customize further
+[compileJava, compileTestJava]*.options*.compilerArgs = ['-Xlint:all,-options']
+
+test {
+	// suppress all console output during testing unless running `gradle -i`
+	logging.captureStandardOutput(LogLevel.INFO)
+	jvmArgs "-javaagent:${configurations.jacoco.asPath}=destfile=${buildDir}/jacoco.exec,includes=*"
```

Comment:
And make this like:
```
jvmArgs "-javaagent:${configurations.jacoco.asPath}=destfile=${buildDir}/jacoco.exec,includes=*",
			"-Dhazelcast.logging.type=slf4j"
```
http://docs.hazelcast.org/docs/latest/manual/html/logging.html
To get rid of logging messages as much as possible. 
It can increase build performance.